pkg/grpc/api_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build unit
+// +build unit
```

Comment:
I think I may have been mistaken when I said you needed this here. I think you just needed the `-tags=unit` in the command to run the test.

I don't think there's a problem with having it, but we should see if we need it.

Reply:
confirmed not needed